src/engineering/judging.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: Judging Service
+author: Raidah Fairooz
+date: July 21 2024
+link: judging
+---
+
+# **Judging Service**
+
+#### 21st July 2024 by Raidah Fairooz
+
+## **Judging Hackathons**
+
+Similar to most competitions, hackathons also require judges to provide teams with valuable feedback and fairly rank projects. Each hackathon organiation decides upon their own judges and the criterion for judging. Depending on the type and tracks of the competition, judges with varied interests and strengths are invited. UCR Hackathons usually have a diverse panel of judges mostly consisting of professors, alumnis, students and/or industry professionals. After hacking ends, every team presents their projects and judges consult on their final selections.
+
+## **Pre Hackathon-Website Judging System**
+
+UCR hackathons used to follow a common system for inviting, assigning and running the judging routine both for in person and online. Organizing committees invited judges and collected all necessary information via email. Due to the number of teams exceeding available judges, we had timed sessions/rounds(~8 minutes) to fairly run the process. Every registered team would be manually assigned to an avaliable judge per round until next rotation. In practice, however, this system caused significant operational delays and often times unfair project evaluations.
+
+Cons:
+
+Issues with judge info collection and verification via email
+
+Added effort and time for organizers to manually assign judges
+
+Bad coordination, rounds with some teams missing judges
+
+Delayed hackathon closing, keeping participants for ~2-3 hours extra
+
+## **Solution 1: Judge Sign Up**
+
+Hackathon website has customized forms for judges to sign up that collects and stores all the necessary information for the admin. Below are all the fields we have the judges fill out. We sort out the judges based on their affiliations and use that data for our system. This highly increased efficiency and reduced possible errors for the hackathon committees.
+
+```
```

Comment:
dont need to show what data we track as its not related to judging service itself